pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md

```diff
@@ -71,8 +71,9 @@ Add a file titled `manifest.json` to your repository with the following code:
 A service worker is a script that your browser runs in the background, 
 separate from a web page, that extends the browsers features by caching requests to improve performance and providing offline functionality. 
 Building a service worker from scratch is possible but time consuming. 
-Libraries like Workbox help, but AMP goes one step further by offering the [AMP Service Worke](https://github.com/ampproject/amp-sw)r, 
-in which AMP automates a lot of steps directly, including the caching of AMP Scripts, assets and documents.
+Libraries like Workbox help, but AMP goes one step further by offering the [AMP Service Worker](https://github.com/ampproject/amp-sw), 
+in which AMP automates a lot of steps directly, including the caching of AMP Scripts, assets and documents
+and implements common best practices such as [navigation preload](https://developers.google.com/web/updates/2017/02/navigation-preload).
```

Comment:
suggestion: s/and implement/as well as implementing/

Reply:
Done! thanks!